.travis.yml

```diff
@@ -4,6 +4,10 @@ node_js:
   - 0.12
   - 4
   - 5
+os:
+  - osx
+  - linux
```

Comment:
Nope. The online example shows osx and linux. So I just tried centos to see if that worked. I know we've been having a couple issues with centos on the generator side of things so I specifically want centos. I may swap out linux for ubuntu and see how that works. I want to play with these settings for a bit before we merge. Once we merge this in here, I'll also add them to the generator project.